usr/share/rear/restore/NBKDC/default/40_restore_backup.sh

```diff
@@ -0,0 +1,16 @@
+LogPrint "
+
+The System is now ready for restore. Please start the restore task from the NovaBACKUP DataCenter Central Management!
```

Comment:
@andyvb070 is there really no way to start the recovery automatic from within rear? Without needing the remote console? In my experience (of those I've done the integration myself) it was always possible (simple or hard).
This is fine as a starter, but is not a perfect automated recovery solution IMHO.